.github/workflows/nightly.yml

```
@@ -43,20 +54,27 @@ jobs:
         run: |
           module load xilinx/vivado
           bazel_tests="$(mktemp)"
-          ./bazelisk.sh query 'attr("tags", "[\[ ]cw310_sival[,\]]", tests(//...))' \
+          ./bazelisk.sh query 'attr("tags", "[\[ ]cw310_sival[,\]]", tests(//sw/device/...))' \
                                                     | grep -v examples \
-                                                    | grep -v third_party \
                                                     | grep -v penetrationtests \
                                                     > "$bazel_tests"
-          ./bazelisk.sh test --build_tests_only --target_pattern_file="$bazel_tests"
+          ./bazelisk.sh test --build_tests_only --target_pattern_file="$bazel_tests" || true
 
       - name: Run tests after ROM_EXT boot stage
         run: |
           module load xilinx/vivado
           bazel_tests="$(mktemp)"
-          ./bazelisk.sh query 'attr("tags", "[\[ ]cw310_sival_rom_ext[,\]]", tests(//...))' \
+          ./bazelisk.sh query 'attr("tags", "[\[ ]cw310_sival_rom_ext[,\]]", tests(//sw/device/...))' \
                                                     | grep -v examples \
-                                                    | grep -v third_party \
                                                     | grep -v penetrationtests \
                                                     > "$bazel_tests"
-          ./bazelisk.sh test --build_tests_only --target_pattern_file="$bazel_tests"
+          ./bazelisk.sh test --build_tests_only --target_pattern_file="$bazel_tests" || true
+
+      - name: Publish bazel test results
+        if: success() || failure()
+        run: |
+          # Bazel produce one xml for each test. So we merge them together.
+          find -L bazel-out -name "test.xml" | xargs merge-junit -o $BAZEL_TEST_RESULTS
+
+          GS_FULL_PATH=$GS_PATH/job/${{ github.job }}/branch/${{ github.ref_name }}/$(date +%Y-%m-%d-%H%M%S)_$BAZEL_TEST_RESULTS
```

```suggestion
          GS_FULL_PATH=$GS_PATH/job/${{ github.job }}/branch/${{ github.ref_name }}/$(date +%Y-%m-%d-%H%M%S)_${BAZEL_TEST_RESULTS}
```
to make it more visible.